### ISSUE TITLE
Invalid numeric argument '/Wextra' for MSVC14

### ISSUE THREAD

**Issue by weiquanmao** [ethz-asl/libnabo#84]
I got error when compiling *.sln with VS2015 on Win 10 
```
cl : Command line error D8021 : invalid numeric argument '/Wextra'
``` 
I found it was caused by the following line:

https://github.com/ethz-asl/libnabo/blob/f96a29264dd4d10b3d213da90da429c1c1fe2112/CMakeLists.txt#L47

which should be defined for GCC.  It was fixed as following in my case.

```
if(CMAKE_COMPILER_IS_GNUCXX)
  add_definitions(-Wall -Wextra)
endif(CMAKE_COMPILER_IS_GNUCXX)
```

**Comment by HannesSommer** [comment]
Thanks for your suggestion. It is a known limitations. Could you maybe test the PR #74 (branch fix/setWarningsCompilerSpecificAndWithCompilerFlags). If it works for you we can merge it and the problem should be solved.

**Comment by weiquanmao** [comment]
It works with [Fix/set warnings compiler specific and with compiler flags ](https://github.com/ethz-asl/libnabo/pull/74/commits/768ca40fa05cfbb62cbebd1c0d245974190a7bde)!

**Comment by HannesSommer** [comment]
Great! Thanks for testing!
This issue should be closed once #74 is merged.

**Comment by simonpierredeschenes** [comment]
#74 was merged.
